javascript/organizing_your_javascript_code/json.md

```diff
@@ -4,17 +4,24 @@ JSON (JavaScript Object Notation) is a standardized format for structuring data.
 
 Fortunately, there isn't much to learn here. We're only including a lesson on it because some formatting rules can cause confusion if you aren't aware of them. Spend 10-15 minutes going through the following resources and you'll be good to go.
 
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- What JSON is and why it is important.
+- Two important JSON methods, `parse` and `stringify`.
+
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
 1. This [JSON MDN tutorial]( https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON) is probably all you need...
-2. Read about the 2 JavaScript methods that you'll most often be using when dealing with JSON - [JSON.parse()](https://www.w3schools.com/js/js_json_parse.asp) and [JSON.stringify()](https://www.w3schools.com/js/js_json_stringify.asp).
-3. Mis-formatted JSON is a common cause of errors. This [JSON formatter website](https://jsonformatter.curiousconcept.com/) lets you paste in JSON code and will search it for formatting errors.
+1. Read about the 2 JavaScript methods that you'll most often be using when dealing with JSON - [JSON.parse()](https://www.w3schools.com/js/js_json_parse.asp) and [JSON.stringify()](https://www.w3schools.com/js/js_json_stringify.asp).
+1. Mis-formatted JSON is a common cause of errors. This [JSON formatter website](https://jsonformatter.curiousconcept.com/) lets you paste in JSON code and will search it for formatting errors.
 
 </div>
 
-### Additional resources
+#### Additional resources
```

Comment:
This should remain a level 3 heading

Reply:
Strange. I changed it back and now markdownlint is complaining 
![image](https://github.com/TheOdinProject/curriculum/assets/154856872/23e0e84e-bc58-4170-a256-53a8a08ca2eb)

Reply:
Ah it's because we don't have a Knowledge check section. Let's keep this as a level 3 heading. We need to create an epic issue for updating knowledge checks throughout the curriculum with the updated verbiage (which will also serve as a means to resolve all lint errors in lessons, rather than updating them bit by bit).

Reply:
I'll leave this open for now if you'd want to add the knowledge check section, though. Will check in again in when I'm off work.

Reply:
I've added the knowledge check section, let me know what you think about it. 

Also, can you please tell me how I can show the changed lines and put it in comment? Like the way you did.
![image](https://github.com/TheOdinProject/curriculum/assets/154856872/917fe05e-6084-4963-a76a-f5d0f314fb58)

Reply:
Step 4 in [commenting on a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) should cover how to go about that